asv_bench/benchmarks/groupby.py

```diff
@@ -71,7 +71,7 @@ def time_groupby_apply_dict_return(self):
 
 class Apply:
     param_names = ["factor"]
-    params = [4, 5]
+    params = [5, 6]
```

Comment:
What's the reason for changing this? 
And if this needs to be changed, also the comments and code below have to be updated (they assume values of 4 and 5)